scripts/examples/50-OpenMV-Boards/51-Pure-Thermal/00-HelloWorld/pure_thermal.py

```diff
@@ -4,36 +4,65 @@
 #
 # Pure Thermal Example Script
 #
-# Thanks for buying the Pure Thermal OpenMV! This example script shows off thermal video
-# overlay onto the color camera image and driving the attached LCD screen and HDMI output.
+# Thanks for buying the Pure Thermal OpenMV! This example script shows
+# off thermal video overlay onto the color camera image and driving
+# the attached LCD screen and HDMI output.
 
-import sensor
+import csi
 import image
 import time
 import display
-import fir
 import math
 import tfp410
 
 
 # Color Tracking Thresholds (Grayscale Min, Grayscale Max)
 threshold_list = [(200, 255)]
 
-sensor.reset()
-sensor.set_pixformat(sensor.RGB565)
-sensor.set_framesize(sensor.WVGA)
+# Set the target temp range here
+min_temp_in_celsius = 20.0
+max_temp_in_celsius = 40.0
+
+csi0 = csi.CSI(cid=csi.OV5640)
+csi0.reset()
+csi0.pixformat(csi.RGB565)
+csi0.framesize(csi.WVGA)
 time.sleep_ms(50)
 
-fir.init(fir.FIR_LEPTON)
-fir_img = image.Image(fir.width(), fir.height(), sensor.GRAYSCALE)
+csi1 = csi.CSI(cid=csi.LEPTON)
+csi1.reset(hard=False)
+csi1.pixformat(csi.GRAYSCALE)
+csi1.framesize(csi.QQVGA)
+
+# Enables exact temperature measurments from the flir lepton.
+# The second argument turns high gain mode on for high temp reading.
+csi1.ioctl(csi.IOCTL_LEPTON_SET_MODE, True, False)
+csi1.ioctl(
+    csi.IOCTL_LEPTON_SET_RANGE, min_temp_in_celsius, max_temp_in_celsius
+)
+
+print(
+    "Lepton Res (%dx%d)"
+    % (
+        csi1.ioctl(csi.IOCTL_LEPTON_GET_WIDTH),
+        csi1.ioctl(csi.IOCTL_LEPTON_GET_HEIGHT),
+    )
+)
+
+print(
+    "Radiometry Available: "
+    + ("Yes" if csi1.ioctl(csi.IOCTL_LEPTON_GET_RADIOMETRY) else "No")
+)
```

Comment:
About these broken lines, the linter allows for a max line of 120 so you don't need to break those like that I think. Also you could use shorter variables, like `min_temp_c`